[inferno-collection]/inferno-april-fools/server.lua

```diff
@@ -9,29 +10,43 @@
 --
 
 --
---		Nothing past this point needs to be edited, all the settings for the resource are found ABOVE this line.
+--		Nothing past this point needs to be edited, all the settings for the resource are found in the config.lua file.
 --		Do not make changes below this line unless you know what you are doing!
 --
 
 local AllPlayers = {}
+local players = {}
 
-RegisterServerEvent('April-Fools:Register')
-AddEventHandler('April-Fools:Register', function()
-    if not AllPlayers[source] then
-        AllPlayers[source] = true
-    end
-end)
+if Config.onConnect then
+    AddEventHandler("playerConnecting", function()
+        local t = os.date("*t")
+        if t.month == Config.onConnectDate.month and t.day == Config.onConnectDate.day then
+            deferrals.defer()
+            if not players[source] then
+                players[source] = true   -- They can reconnect
+                CancelEvent()
+                deferrals.done("You have been permanently banned from this server. You can view the reason and appeal this ban at " + Config.url)
```

Comment:
Lua needs `..` not `+`, or it tries to treat it as an int instead of a string 😛 
```suggestion
                deferrals.done("You have been permanently banned from this server. You can view the reason and appeal this ban at " .. Config.url)
```